src/Symfony/Bundle/TwigBundle/TwigEngine.php

```diff
@@ -51,7 +51,8 @@ public function setDefaultEscapingStrategy($strategy)
     }
 
     /**
-     * @deprecated Deprecated since version 2.3, to be removed in 3.0. Use TwigDefaultEscapingStrategy instead.
+     * @deprecated since version 2.3, to be removed in 3.0.
```

Comment:
actually, this message should mention the patch version of 2.3, because TwigDefaultEscapingStrategy does not exist in 2.3.0. It was added in a recent patch version to fix a bug.